source/usage-examples/updateOne.txt

```diff
@@ -4,35 +4,38 @@ Update a Document
 
 .. default-domain:: mongodb
 
-You can update a single document using the :node-api:`updateOne()
-<Collection.html#updateOne>`
-method. ``updateOne()`` accepts a filter document, and updates the first
-document that matches the filter using a provided update document. The
-update document requires an :manual:`Update Operator
-</reference/operator/update/#update-operators>` to modify a field in a
-document.
-
-Create an :mdn:`Object
-<Web/JavaScript/Reference/Global_Objects/Object>` to specify additional
-options. Set the ``upsert`` option to ``true`` to create a new document
-if no documents match the filter.
+You can update a single document using the :node-api:`collection.updateOne()
+<Collection.html#updateOne>` method. ``updateOne()`` accepts a filter
+document and a replacement document. If the query matches documents in the
+collection, it replaces the matches with the replacement document. The
```

Comment:
This sentence probably requires some significant restructuring to work in the context of an update document rather than a replacement document. Something that specifies fields & values, perhaps? At the very least we need to move away from the word "replace" :smile:

Reply:
Makes sense. I'll _update_ it.